docs/api/push-notifications.md

```diff
@@ -0,0 +1,50 @@
+# pushNotifications
+
+> Register for remote push notification services
+
+Process: [Main](../glossary.md#main-process)
```

Comment:
I think this document would benefit from a short example explaining how these methods and events fit together.